src/components/_global/BalDropdown/BalDropdown.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+
+/**
+ * TYPES
+ */
+type Props = {
+  options: Array<any>;
```

Comment:
This is a good start, I guess in the future we could start typing it into something that has a `label` and `value` and then the `selected` logic can live here (so that we don't re-implement it in each dropdown).

For customization, we could have a `formatOptionLabel`, etc.